src/js/index.js

```diff
@@ -1,26 +1,28 @@
 // Component imports
-import { defaultComp } from './components/default';
+import { defaultComp } from "./components/default";
```

Comment:
For js files, it's recommended to use single quotes instead of double-quotes. Please replace this where needed

Reply:
Resolved